code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -595,16 +597,64 @@
 	return "[rank] [pick(beggings)][pick(endings)]"
 
 /obj/effect/mob_spawn/human/pirate/Destroy()
-	new/obj/structure/showcase/machinery/oldpod/used(drop_location())
+	if(spawn_oldpod)
+		new /obj/structure/showcase/machinery/oldpod/used(drop_location())
 	return ..()
 
 /obj/effect/mob_spawn/human/pirate/captain
-	rank = "Captain"
-	outfit = /datum/outfit/pirate/space/captain
+	rank = "Renegade Leader"
+	outfit = /datum/outfit/pirate/captain
 
 /obj/effect/mob_spawn/human/pirate/gunner
+	rank = "Rogue"
+
+/obj/effect/mob_spawn/human/pirate/skeleton
+	name = "pirate remains"
+	desc = "Some unanimated bones. They feel like they could spring to life any moment!"
+	random = TRUE
+	density = FALSE
+	icon = 'icons/effects/blood.dmi'
+	icon_state = "remains"
+	spawn_oldpod = FALSE
+	mob_name = "a space pirate"
+	mob_species = /datum/species/skeleton
+	outfit = /datum/outfit/pirate
+	rank = "Mate"
+
+/obj/effect/mob_spawn/human/pirate/skeleton/captain
+	rank = "Captain"
+	outfit = /datum/outfit/pirate/captain
+
+/obj/effect/mob_spawn/human/pirate/skeleton/gunner
 	rank = "Gunner"
 
+/obj/effect/mob_spawn/human/pirate/silverscale
+	name = "elegant sleeper"
+	desc = "Cozy. You get the feeling you aren't supposed to be here, though..."
+	random = TRUE
+	icon = 'icons/obj/machines/sleeper.dmi'
+	icon_state = "sleeper"
+	mob_name = "a space pirate"
+	mob_species = /datum/species/lizard/silverscale
+	outfit = /datum/outfit/pirate/silverscale
+	rank = "High-born"
+
+/obj/effect/mob_spawn/human/pirate/silverscale/generate_pirate_name()
+	var/first_name
+	if(gender == MALE)
```

Comment:
Change to a switch/case that has one for MALE, FEMALE, and neither (in which case you'd pick from male + female).